api/metaschemas/views.py

```diff
@@ -50,11 +51,17 @@ class MetaSchemasList(JSONAPIBaseView, generics.ListAPIView):
     view_category = 'metaschemas'
     view_name = 'metaschema-list'
 
+    # implement ODMFilterMixin
+    def get_default_odm_query(self):
+        return (
+            Q('name', 'in', ACTIVE_META_SCHEMAS) &
+            Q('schema_version', 'eq', LATEST_SCHEMA_VERSION) |
+            Q('name', 'eq', 'Preprint Provider')
```

Comment:
Hmmm. I don't like that. Not your fault, I think it's the old thing that I don't like, but your addition brings out the weirdness. This will probably be easy to fix in Django when we have real SQL. But there's not, for example, a 'latest_version' property or anything useful like that on the metaschemas as things stand?